libr/io/p/io_mach.c

```diff
@@ -197,32 +188,30 @@ static ut64 getNextValid(RIO *io, RIODesc *fd, ut64 addr) {
 	return lower;
 }
 
-static int __read(RIO *io, RIODesc *desc, ut8 *buf, int len) {
+static int __read(RIO *io, RIODesc *fd, ut8 *buf, int len) {
 	vm_size_t size = 0;
 	int blen, err, copied = 0;
 	int blocksize = 32;
-	RIODescData *dd = (RIODescData *)desc->data;
-	if (!io || !desc || !buf || !dd) {
-		return -1;
-	}
-	if (dd ->magic != r_str_hash ("mach")) {
+	RIOMach *riom;
+	if (!io || !fd || !buf || !riom) {
```

Comment:
Uninitialezed riom regression

Reply:
read the rest of the code, this line is fine